boards/arm/stm32f746g_disco/pinmux.c

```diff
@@ -35,6 +38,20 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PG13, STM32F7_PINMUX_FUNC_PG13_ETH},
 	{STM32_PIN_PG14, STM32F7_PINMUX_FUNC_PG14_ETH},
 #endif /* CONFIG_ETH_STM32_HAL */
+#ifdef CONFIG_I2C_1
+	{STM32_PIN_PB8, STM32F7_PINMUX_FUNC_PB8_I2C1_SCL},
+	{STM32_PIN_PB9, STM32F7_PINMUX_FUNC_PB9_I2C1_SDA},
+#endif  /* CONFIG_I2C_1 */
+#ifdef CONFIG_SPI_2
+	{STM32_PIN_PI0,  STM32F7_PINMUX_FUNC_PI0_SPI2_NSS},
+	{STM32_PIN_PI1,  STM32F7_PINMUX_FUNC_PI1_SPI2_SCK},
```

Comment:
Is STM32F7_PINMUX_FUNC_PI1_SPI2_SCK declared in drivers/pinmux/stm32/pinmux_stm32f7.h?

Reply:
Yes, i added all SPI2 pinmux definitions to this file.